LibraBFT/Impl/Consensus/RoundManager/PropertyDefs.agda

```diff
@@ -128,3 +178,32 @@ substVoteCorrect refl refl refl refl refl refl (mkVoteCorrect (mkVoteCorrectInv
   mkVoteCorrect (mkVoteCorrectInv round≡ postLv≡) (Left (mkVoteCorrectOld lvr≡ lv≡))
 substVoteCorrect refl refl refl refl refl refl (mkVoteCorrect (mkVoteCorrectInv round≡ postLv≡) (Right (mkVoteCorrectNew epoch≡ lvr< postLvr≡))) =
   mkVoteCorrect (mkVoteCorrectInv round≡ postLv≡) (Right (mkVoteCorrectNew epoch≡ lvr< postLvr≡))
+
+record VoteMsgOutsCorrect (pre post : RoundManager) (outs : List Output) (epoch : Epoch) (round : Round) : Set where
+  constructor mkVoteMsgOutsCorrect
+  field
+    vm  : VoteMsg
+    pid : Author
+    voteMsgOuts : VoteMsgOuts outs vm (pid ∷ [])
+    voteCorrect : VoteCorrect pre post epoch round (vm ^∙ vmVote)
```

Comment:
I am going to re-organize this file, both so that the various property definitions and lemmas are more easily located and so that the names are more descriptive.